### ISSUE TITLE
Rat cap 3D models should be updated

### ISSUE THREAD

**Issue by jiumao2** [buzsakilab/3d_print_designs#1]
We tried the rat cap 3D models and found that the rat_cap_top did not fit the M2 thumb screws.

We bought the rat cap 3D models from 3DNeuro and that rat_cap_top top performed well. There must be an updated version of this 3D model.

I would appreciate it if you could update the models on the GitHub.

**Comment by petersenpeter** [comment]
Hi Yue

We will look into this. @misiVoroslakos can you help here? :-)

**Comment by jiumao2** [comment]
Hi Petersen,

Any updates on this topic?

**Comment by misiVoroslakos** [comment]
Hi Yue,

According to the naming convention, the proper top is uploaded to the Github. I am printing it now, to make sure that the holes are the correct size. Meanwhile, could you please let us know which printer, resin and resolution you tried to print this?

**Comment by misiVoroslakos** [comment]
Hi Yue,

My print has finished (FormLabs, Form2 printer, clearv04 resin, 0.05 mm layer thickness) and the thumb screw fits here. Did you experience some issues with the nut or the front locking mechanism?
 
![IMG_4400](https://github.com/buzsakilab/3d_print_designs/assets/39443662/0cf4565f-6dd2-46cf-ab08-49ae5ed68d68)

**Comment by jiumao2** [comment]
Hi misiVoroslakos,

This one is from 3DNeuro:
![top_1](https://github.com/buzsakilab/3d_print_designs/assets/62061407/de391e52-9516-4402-827d-8879bc3bf939)

This one is our print (we don't have a good printer and order the print from a company):
![top_2](https://github.com/buzsakilab/3d_print_designs/assets/62061407/2a309b2b-01cc-4bd2-9ccb-5969c53bd4dc)

I saw a little difference between these two prints. So, I wondered whether you have updated your model to make the screw fit. 

The other parts of my print performed well, and we adapted the top by widening the hole after printing. Maybe we should try smaller thumb screws as yours.

Thank you very much for your support!
